virocon/distributions.py

```diff
@@ -1347,3 +1347,158 @@ def _fit_mle(self, sample):
 
     def _fit_lsq(self, data, weights):
         raise NotImplementedError()
+ 
+class VonMisesDistribution(Distribution):
+    """
+    A Von Mises (Circular Norm) Distribution. 
+    
+    The distributions probability density function is given by [1]_: 
+    
+    :math:`f(x) = \\frac{\\exp{k \\cos{x - \\mu}}}{2 \\pi I_0(k) \\sigma}`
```

Comment:
Would you mind using `\\kappa` instead of k here? Just to make sure the used latex compiler does not choke on Unicode chars.